nuget/lib/dependabot/nuget/file_parser/project_file_parser.rb

```diff
@@ -131,25 +169,18 @@ def add_transitive_dependencies(project_file, doc, dependency_set)
         end
 
         def add_transitive_dependencies_from_project_references(project_file, doc, dependency_set)
-          project_file_directory = File.dirname(project_file.name)
-          is_rooted = project_file_directory.start_with?("/")
-          # Root the directory path to avoid expand_path prepending the working directory
-          project_file_directory = "/" + project_file_directory unless is_rooted
-
           # Look for regular project references
-          doc.css(PROJECT_REFERENCE_SELECTOR).each do |reference_node|
+          project_refs = doc.css(PROJECT_REFERENCE_SELECTOR)
+          # Look for ProjectFile references (dirs.proj)
+          project_files = doc.css(PROJECT_FILE_SELECTOR)
```

Comment:
This scenerio doesn't seem to be affecting the repo I was testing against for some reason, but it's possible for the system to ignore dirs.proj and just run the projects under it individually because it didn't pick up any of their dependencies.

Reply:
Do dirs.proj files make it through this bit of code or do we "evaluate" the dirs.proj to get a list of referenced .csproj files?

Reply:
dirs.proj files make it through here, and we do call NugetUpdater.CLI directly on dirs.proj files. NugetUpdater.CLI then finds all the csproj's an checks them for the dependency we want to update.